contact-style.css

```diff
@@ -0,0 +1,41 @@
+.keepintouch {
+  background-color: #252525;
```

Comment:
use camelCase for multi word classes `keepInTouch`
it's easier to read. you can also do `keep-in-touch`

Reply:
Thank you so much Juan for all of your feedback and suggestions! It truly means a lot.